### PR TITLE
test all downstreams.

### PR DESCRIPTION
**Pull request by n3wscott**
Drops eventing-contrib, adds everything else. 😎

### CODE REVIEW

File: .github/workflows/knative-downstream.yaml [hunk 1]
```diff
@@ -20,211 +20,90 @@ on:
...
-        platform: [ubuntu-latest]
-
-    runs-on: ${{ matrix.platform }}
-
+        downstream: # only for github.com/knative
+          - serving
+          - networking
+          - eventing
+          - operator
+          - client
```
Comment (mattmoor) [inline .github/workflows/knative-downstream.yaml:33]:
```suggestion
        include:
          - downstream: serving
             org: knative
          - downstream: networking
             org: knative
          - downstream: eventing
             org: knative
          - downstream: operator
             org: knative
          - downstream: client
             org: knative
```

Comment (n3wscott) [reply .github/workflows/knative-downstream.yaml]: rad.

**Comment by mattmoor** [comment]
Can we wait until they're passing...? 😉

**Comment by n3wscott** [comment]
> Can we wait until they're passing...? 😉

they might not... that is kinda the point

**Comment by mattmoor** [comment]
The set of things that run presubmit here should be expected to stay green the vast majority of the time.

If things aren't green, we should remove them here.  I don't think 11 failing legs is where we expect the steady state to be, so let's get what passes passing and comment out the rest.

**Comment by n3wscott** [comment]
@mattmoor It looks like all the current failures are related to those deps not being in sync with networking or eventing, which will get resolved at 6pm

**Comment by mattmoor** [comment]
@n3wscott should I run it now?  it's a click away 😇

**Pull request merged**